[Focus Mode] New DevTools overview docs and screenshots (#18)
captainbrosset/edge-developer: microsoft-edge/devtools-guide-chromium/about-tools.md

<!-- ====================================================================== -->
## The More Tools menus

The **More Tools** (**+**) menu in the **Activity Bar** and in the **Quick View** toolbar is dynamic: it omits any tab tools that are open on that toolbar.

<!-- ====================================================================== -->

> ## Activity Bar tools vs. Quick View tools

Reviewer: Hmm ~ Are we using Activity Bar tools and Panel tools interchangeably?  Feel like maybe we should stick with one? The Command Menu still labels some views as "Panel" which is confusing.

Reviewer 2: gh rendered section:
https://github.com/captainbrosset/edge-developer/blob/user/pabrosse/focus-mode-main/microsoft-edge/devtools-guide-chromium/about-tools.md#activity-bar-tools-vs-quick-view-tools

Reviewer 2: @captainbrosset - add "Show" after > prompt, in the png.  if don't type "Show", the [Panel] commands aren't immediately followed by the [Quick View] commands, but there are many green [Performance] and [Persistence] comments in between them.  Here's what i see, only _after_ typing "Show":

![image](https://github.com/captainbrosset/edge-developer/assets/45407486/250d96ca-99fa-4e31-942e-8aead67ace73)

Author: Yes, you're right, typing "Show" is needed, I'll it to the png file.

@JLSeto **Activity Bar** and **Panel** mean 2 different things. But I'm not super happy with the fact that our docs uses these 2 terms. It does feel interchangeable, yes, but they're in fact used to talk about 2 different (although closely related) things.

In the current docs, we use **Main toolbar** and **Panel**. So in this PR I'm only changing **Main toolbar** to **Activity Bar**, but I'm keeping **Panel**.

The **Activity Bar** is really just the toolbar at the top that contains that tabs with icons, and the ... and ? icons, as well as the inspect and device emulation tool. That's what we used to call the **Main toolbar**.
But then, the tools that appear right below this toolbar, when you click on a tool tab is what we call a **Panel**.
The little label in the Command Menu tells you whether a tool is a **Panel** tool (as in, whether it opens in the **Activity Bar**) or a **Quick View** tool (as in, whether it opens in the **Quick View**).

Maybe we can rephrase this article to make things clearer. I kind of would love @mikehoffms' help on this to be honest.

Reviewer: Oh I see - that makes sense.  I think I got confused with the two diffs since it shows 
`Panel Tools -> Activity Bar tools`
`Drawer Tools -> Quick View tools`

So maybe it should be kept as `Panel tools vs. Quick View tools`?

Reviewer 2: I plan to inspect this section of the article in VS Code and maybe polish the term usage and push a commit.  I'll post status here.

Reviewer 2: fixed locally

In the **Command Menu**:

* _Panel tools_ are tools which open in the **Activity Bar** by default.

* _Quick View tools_ are tools which open in the **Quick View** toolbar by default, at the bottom of DevTools.  Press **Esc** to show or hide the **Quick View**.

> The **Command Menu** first lists the Panel tools, and then the **Quick View** tools:

Reviewer 2: ```suggestion
The **Command Menu** first lists the **Activity Bar** tools (indicated by the **Panel** label), and then the **Quick View** tools:
```

![The Command Menu, showing the Panel tools grouped together, and then the Quick View tools](./about-tools-images/command-menu-panel-vs-drawer-tools.png)

To move a tool to the other toolbar, right-click the tool's tab and then select **Move to bottom Quick View** or **Move to top Activity Bar**:

> ![The contextual menu for the Changes tool, which is displayed in the Quick View toolbar. The menu contains the Move to top Activity Bar item](./about-tools-images/move-from-quickview-to-activitybar.png)

Reviewer 2: ```suggestion
![The right-click menu for a tool in the Quick View toolbar, including the 'Move to top Activity Bar' menu item](./about-tools-images/move-from-quickview-to-activitybar.png)
```
